packages/api/models/granules.js

```diff
@@ -263,55 +253,6 @@ class Granule extends Manager {
     await Lambda.invoke(process.env.invoke, lambdaPayload);
   }
 
-  /**
-   * Move a granule's files to destinations specified
-   *
-   * @param {Object} g - the granule record object
-   * @param {Array<{regex: string, bucket: string, filepath: string}>} destinations
-   *    - list of destinations specified
-   *    regex - regex for matching filepath of file to new destination
-   *    bucket - aws bucket of the destination
-   *    filepath - file path/directory on the bucket for the destination
-   * @param {string} distEndpoint - distribution endpoint URL
-   * @returns {Promise<undefined>} undefined
-   */
-  async move(g, destinations, distEndpoint) {
```

Comment:
Explicitly removing this model method as in the postgres release we do not want this behavior available on the API model